### PR TITLE
Remove ngMaterial

### PR DESCRIPTION
**Pull request by kennethlynne**

### CODE REVIEW

File: src/ui/states/main/main.controller.spec.js [hunk 1]
```diff
@@ -29,9 +29,9 @@ describe('states: main', function () {
       rootScope.$digest();
     });
 
-    it('should define more than 5 things', inject(function () {
+    it('should expose things on scope', inject(function () {
       expect(angular.isArray(scope.things)).toBeTruthy();
-      expect(scope.things.length > 1).toBeTruthy();
+      expect(scope.things).toEqual(things);
```
Comment (AlmaasAre) [inline src/ui/states/main/main.controller.spec.js]: Goodie!

File: src/ui/states/main/main.html [hunk 2]
```diff
@@ -1,34 +1,17 @@
...
+<div>
 
   <header>
     <navbar-component></navbar-component>
   </header>
 
-  <md-content>
-    <section class="jumbotron">
-      <h1>Welcome friend!</h1>
+  <h1>Welcome to Sweden, here is your complimentary horse!</h1>
```
Comment (AlmaasAre) [inline src/ui/states/main/main.html]: :horse:  :laughing:

**Comment by AlmaasAre** [comment]
Må bare fikse e2e-testene nå så blir det schweet!

**Comment by kennethlynne** [comment]
On it

man. 13. apr. 2015, 14:59 skrev Are Almaas <notifications@github.com>:

**Comment by AlmaasAre** [comment]
@kennethlynne i just mrged it, hope is ok

**Pull request merged**